modules/nf-scil/registration/antsapplytransforms/meta.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: "registration_antsapplytransforms"
+description: register image from transform
+keywords:
+  - MRI
+  - registration
+  - medical imaging
+tools:
+  - "ANTs":
+      description: "perform registration using transform as input"
+      homepage: "https://antsx.github.io/ANTsRCore/reference/antsApplyTransforms.html"
```

Comment:
Add what I put in my PR for the meta file with the documentation and the paper please

https://github.com/scilus/nf-scil/pull/86/files#diff-d35a1509614d4208dbfe5f63e98176dbd74574ad6523c4a7f589f339118cd022